poutyne/framework/model.py

```diff
@@ -321,11 +321,11 @@ def fit_dataset(self,
                     validation_steps=None,
                     batches_per_step=1,
                     initial_epoch=1,
-                    verbose=True,
-                    progress_options=None,
-                    callbacks=None,
                     num_workers=0,
                     collate_fn=None,
+                    verbose=True,
+                    progress_options: Union[dict, None] = None,
+                    callbacks=None,
```

Comment:
Not sure why you moved that there. I put num_workers and collate_fn with dataloader_kwargs because they're all arguments for the dataloader.

Reply:
I've tried to standardize the order of the args.